docs/GraphQL-API/mutations/price.md

```diff
@@ -0,0 +1,93 @@
+# Price mutation examples
+
+Price object has three mutations:
+
+- `createPrice`
+- `updatePrice`
+- `deletePrice`
+
+## `createPrice`
+
+This mutation creates a new price. You must pass the `priceType` (GID) field to set the price type ID.
+
+```gql
+mutation createPrice($input: CreatePriceInput!) {
+  createPrice(input: $input) {
+    price {
+      id
+      name
+    }
+  }
+}
+```
+
+### Query variables for `createPrice`
```

Comment:
Maybe mutation variables ?

Reply:
They are still called query variables but yes we can name them as mutation variables.